src/app/Helpers/Import.php

```diff
@@ -61,7 +63,7 @@ public static function getLatestScrapedFile($prefix, $all_states = false) {
     }
 
     public static function getMatchedLocations($prefix, $since = null, $one_match_only = true) {
-        $data = static::getLatestScrapedFile($prefix);
+        $data = static::getLatestImportFile($prefix);
```

Comment:
I assume that this is the only place that this refactoring needs to happen. I `grep`d and can confirm that is the case. Consider this comment for future us to use to jog our memories.